nuitka/freezer/Onefile.py

```diff
@@ -170,13 +170,19 @@ def packDistFolderToOnefileLinux(onefile_output_filename, dist_dir, binary_filen
     stdout_file = openTextFile(stdout_filename, "wb")
     stderr_file = openTextFile(stderr_filename, "wb")
 
+    if options.appimage_compression == "gzip":
```

Comment:
For options, we add an accessor function, like `getAppImageCompression()` for use here.

Reply:
I added one in Options.py, I take it that was the correct location?